lib/services/ios-project-service.ts

```diff
@@ -145,8 +145,14 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 						!!buildOptions.buildForDevice ||
 						!!buildOptions.buildForAppStore;
 					if (forDevice) {
+						const ipaFileName = _.find(
+							this.$fs.readDirectory(
+								this._platformData.getBuildOutputPath(buildOptions)
+							),
+							(directory) => path.extname(directory) === ".ipa"
```

Comment:
```suggestion
							(entry) => path.extname(entry) === ".ipa"
```
Minor suggestion, since we are not strictly dealing with directories.